### PR TITLE
feat: break out s3 image expiration time to env variable

### PR DESCRIPTION
**Pull request by gictor**
### Description:
Break out expiration time to an environment variable

### CODE REVIEW

File: docs/docs/guides/enable-cloud-storage.mdx [hunk 1]
```diff
@@ -27,11 +27,12 @@ Create a new bucket with the following details:
 - disable `enforce public access` and select `fine-grained`
 - Protection: `none`
 
-Go to setings > Interoperability and create a `Access key for service account`
+Go to settings > Interoperability and create a `Access key for service account`
 
 - Copy the `access key` and `secret`
 
 `S3_ENDPOINT` for google is `https://storage.googleapis.com` 
+`S3_REGION` for google is `auto`
```
Comment (rephus) [inline docs/docs/guides/enable-cloud-storage.mdx:35]: Cool

File: packages/backend/src/config/parseConfig.ts [hunk 2]
```diff
@@ -296,6 +297,7 @@ const mergeWithEnvironment = (config: LightdashConfigIn): LightdashConfig => {
             secretKey: process.env.S3_SECRET_KEY,
             bucket: process.env.S3_BUCKET,
             endpoint: process.env.S3_ENDPOINT,
+            expirationTime: process.env.S3_EXPIRATION_TIME,
```
Comment (rephus) [inline packages/backend/src/config/parseConfig.ts:300]: nice !

**Review by rephus — APPROVED** [review]
Looks good to me, thanks for your contribution !

**Review by rephus — CHANGES_REQUESTED** [review]
One of the tests are failing because it is not formatted properly... did you run our commit hooks ? 

If you can't do it, I'll have to create a separate branch to merge this

image

**Comment by rephus** [comment]
@all-contributors add @gictor  for code

**Comment by gictor** [comment]
> One of the tests are failing because it is not formatted properly... did you run our commit hooks ?
> 
> If you can't do it, I'll have to create a separate branch to merge this
> 
> image

You are correct! I think I got it right now :)

**Review by rephus — APPROVED** [review]

**Pull request merged**